glusterd2/commands/volumes/volume-start.go

```diff
@@ -76,7 +77,8 @@ func volumeStartHandler(w http.ResponseWriter, r *http.Request) {
 	volname := mux.Vars(r)["volname"]
 	var req api.VolumeStartReq
 
-	if err := restutils.UnmarshalRequest(r, &req); err != nil {
+	// request body is optional
+	if err := restutils.UnmarshalRequest(r, &req); err != nil && err != io.EOF {
```

Comment:
just for clarification: if Unmarshal fails, will `req.Force` gets default value `false`?

Reply:
Yes